### ISSUE TITLE
[security] Remove uploaded documents once the embedding is done

### ISSUE THREAD

**Issue by leobenkel** [open-webui/open-webui#4209]
**Is your feature request related to a problem? Please describe.**

Today, the back-end keep the documents uploaded. And for safety, that's not super clean.

**Describe the solution you'd like**

Would be nice to have the document uploaded, with obfuscated name. The embedding is done. Then the file is removed, no trace left.

**Describe alternatives you've considered**

Right now, I have to remove the file manually which is annoying.

**Comment by leobenkel** [comment]
why the closing without conversation ?

**Comment by justinh-rahb** [comment]
@leobenkel we close issues that lack sufficient details for reproduction or resolution, or that don't follow our issue/feature request template. This helps us maintain clarity and ensures we can address valid concerns effectively. Thank you for understanding!

**Comment by leobenkel** [comment]
> @leobenkel we close issues that lack sufficient details for reproduction or resolution, or that don't follow our issue/feature request template. This helps us maintain clarity and ensures we can address valid concerns effectively. Thank you for understanding!

I followed the predefined pattern for feature request, what did i miss ?
